pkg/deploy/graph/analysis/dc.go

```diff
@@ -124,3 +131,92 @@ Node:
 
 	return markers
 }
+
+func FindPersistentVolumeClaimWarnings(g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
+	markers := []osgraph.Marker{}
+
+	for _, uncastDcNode := range g.NodesByKind(deploygraph.DeploymentConfigNodeKind) {
+		dcNode := uncastDcNode.(*deploygraph.DeploymentConfigNode)
+		marker := pvcMarker(g, f, dcNode)
+		if marker != nil {
+			markers = append(markers, *marker)
+		}
+	}
+
+	return markers
+}
+
+func pvcMarker(g osgraph.Graph, f osgraph.Namer, dcNode *deploygraph.DeploymentConfigNode) *osgraph.Marker {
+	for _, uncastPvcNode := range g.SuccessorNodesByEdgeKind(dcNode, deployedges.VolumeClaimEdgeKind) {
+		pvcNode := uncastPvcNode.(*kubegraph.PersistentVolumeClaimNode)
+
+		if !pvcNode.Found() {
+			return &osgraph.Marker{
+				Node:         dcNode,
+				RelatedNodes: []graph.Node{uncastPvcNode},
+
+				Severity: osgraph.WarningSeverity,
+				Key:      MissingPVCWarning,
+				Message:  fmt.Sprintf("%s points to a missing persistent volume claim (%s).", f.ResourceName(dcNode), f.ResourceName(pvcNode)),
+				// Suggestion: osgraph.Suggestion(fmt.Sprintf("oc create pvc ...")),
```

Comment:
Mark with `//TODO`.  I'm assuming we'll come back to this once we have the command to create this?